### PR TITLE
Fix bug on emacs 25: complete arguments for `setq` function

### PR DESCRIPTION
**Pull request by gfZeng**

### CODE REVIEW

File: fcitx.el [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ;; Author: Junpeng Qiu <qjpchmail@gmail.com>
 ;; Keywords: extensions
+;; Package-Version: 20160103.2101
```
Comment (xuchunyang) [inline fcitx.el]: This is not wanted.

**Comment by xuchunyang** [comment]
It looks good to me for the rest.  (BTW, setq is not really a function, though this doesn't matter here.)

**Comment by gfZeng** [comment]
@xuchunyang fixed

**Comment by cute-jumper** [comment]
Thanks @gfZeng @xuchunyang !

**Pull request merged**